apiserver/test/e2e/resources/counter_sample.py

```diff
@@ -0,0 +1,29 @@
+
+import ray
```

Comment:
This is very similar to https://github.com/ray-project/kuberay/blob/master/apiserver/test/job/code.yaml. Is there any way to merge the 2?

Reply:
Not really, and I think they can be separate.